src/ray/gcs/gcs_server/gcs_placement_group_manager.cc

```diff
@@ -300,8 +300,15 @@ void GcsPlacementGroupManager::OnPlacementGroupCreationSuccess(
   auto placement_group_id = placement_group->GetPlacementGroupID();
   RAY_CHECK_OK(gcs_table_storage_->PlacementGroupTable().Put(
       placement_group_id, placement_group->GetPlacementGroupTableData(),
-      [this, placement_group_id](Status status) {
+      [this, placement_group_id, placement_group](Status status) {
         RAY_CHECK_OK(status);
+        // Update the resource in gcs resource manager
```

Comment:
Btw, any reason why we are doing it all at once instead of when each bundle creation request is replied from raylet? Any pros and cons in your mind?

Reply:
The main issue I can imagine is that it will have unsycned states. For example, raylet will contain a bunch of pg resources although they are not updated in GCS

Reply:
Should that be OK? I mean, for example, if the commit failed, we'll revert it back, right? I put it here is because that's the place where the commit happens.
I feel either way probably is ok. 
Is anything broken come to your mind about "raylet will contain a bunch of pg resources although they are not updated in GCS" ?